src/mockData/userFavorites.js

```diff
@@ -45,4 +45,29 @@ export const mockUserFavesArray = [
     "vote_average": "6.9",
     "overview": "In a series of escalating encounters, security guard David Dunn uses his supernatural abilities to track Kevin Wendell Crumb, a disturbed man who has twenty-four personalities. Meanwhile, the shadowy presence of Elijah Price emerges as an orchestrator who holds secrets critical to both men."
   }
+]
+
+export const mockUserFavesArrayAfterReducer = [
```

Comment:
Good addition here! I like that all of our mocks have very clear names of what they represent, even if they get long. In this case I feel it's better to be explicit. 👍